lib/sacastats_web/templates/character/weapons.html.heex

```diff
@@ -104,7 +104,7 @@
                                         </span>
                                     </label>
                                 </div>
-                                <%= for faction <- @character["factions"] do %>
+                                <%= for faction <- ["NC", "TR", "VS", "NS", "NSO"] do %>
```

Comment:
Do you think it would be best to have this here or have like a `get_faction_aliases()` method in the `character_controller.ex`

Reply:
I figured since it was such a small list that we could have it inline, but maybe I can add the aliases to `SacaStats.factions/0`?

Reply:
I think that it might be a good idea to add if we ever end up changing stuff with it like having ns and nso be the same thing or something like that later down the line.

Reply:
Cool, just pushed these changes :+1: